packages/ts/react-crud/src/autocrud.obj.css

```diff
@@ -46,10 +40,19 @@
 
 .auto-crud .auto-form {
   width: 40%;
-  border: solid 1px var(--lumo-contrast-20pct);
-  border-left-width: 0;
-  box-shadow: var(--lumo-box-shadow-s);
+  min-width: 300px;
```

Comment:
Also use a min-width for the form panel. The current value ensures the form's toolbar doesn't overflow when all three buttons are visible.